apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -56,18 +65,11 @@ private async Task<int> InsertAsync(SpannerParameterCollection values)
             record[_lastKey] = bytes;
         }
 
-        private async Task WithLastRowAsync(Action<SpannerDataReader> readerAction)
+        internal static async Task WithLastRowAsync(Action<SpannerDataReader> readerAction, SpannerConnection connection, Func<SpannerConnection, Task<SpannerDataReader>> getReader)
```

Comment:
Should this be as follows, or am I missunderstanding?
```suggestion
        internal static async Task WithLastRowAsync(Action<SpannerDataReader> lastRowAction, SpannerConnection connection, Func<SpannerConnection, Task<SpannerDataReader>> getReader)
```